setup.cfg

```diff
@@ -19,7 +19,6 @@ classifiers =
     Topic :: Internet :: WWW/HTTP
     Topic :: Software Development :: Libraries :: Python Modules
     Programming Language :: Python
-    Programming Language :: Python :: 3.8
     Programming Language :: Python :: 3.9
```

Comment:
Should this line also be removed?

Reply:
Yes, as this change is removing support for Python 3.8.

Reply:
Oh nvm I had an off-by-one error